dask/dataframe/core.py

```diff
@@ -4687,6 +4688,8 @@ def apply_concat_apply(
         as is.
     split_out_setup_kwargs : dict, optional
         Keywords for the `split_out_setup` function only.
+    sort : bool, default True
```

Comment:
The default up above is None. Which do we want?

Reply:
I'm thinking we would want the default to be `None` - In which case we would let the backend decide the default, but I'm not sure yet.  Perhaps I will need to revisit this question after I have added some tests